### PR TITLE
add some style

### PR DESCRIPTION
**Pull request by ungoldman**
Cloned down your project to fiddle with it locally and decided to try adding a little style -- hope you don't mind! Here's what it looks like:

screen shot 2017-02-07 at 5 24 24 pm

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  background: #eee;
+  border-color: #777;
+  color: #222;
+}
+#pad {
+  background: #fff;
+  border: 1px solid #ccc;
+  color: #333;
+  font-size: 16px;
+  flex-grow: 1;
```
Comment (ungoldman) [inline style.css:71]: using flexbox to set the height of the textarea -- if support for older browsers is a concern you may not want to do it this way

**Comment by hackergrrl** [comment]
Wow! This looks super slick -- thank you so much Nate! :tada:

**Comment by hackergrrl** [comment]
Now live! http://pad.eight45.net/?doc=hello-noffle

**Comment by ungoldman** [comment]
Yay!

**Pull request merged**